tests/test_source_check.py

```
@@ -1,162 +1,205 @@
+            ('p2', 't3', 'd1'),
+            ('p1', 't4', 'd1'),
+            ('s1', 't1', 'p1'),
+            ('s2', 't5', 'd2'),
+        },
+        set(),
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    'test_id, start_nodes, end_nodes, connections, expected_good, expected_demand_orphans, expected_other_orphans',
```

Ruff (0.14.1)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)